packages/core/src/ObjectContainer.js

```diff
@@ -361,11 +361,11 @@ export default class ObjectContainer {
   get(name) {
     let entry = this._getEntry(name);
 
-    if (entry.sharedInstance === null) {
+    if (entry?.sharedInstance === null) {
       entry.sharedInstance = this._createInstanceFromEntry(entry);
     }
 
-    return entry.sharedInstance;
+    return entry?.sharedInstance;
```

Comment:
The optional chaining is not neede in both cases since the `sharedInstance` is always defined.

Reply:
The optional chaining is needed in both cases as the entry can be null if its not found.